pages/volunteer/Volunteer.js

```diff
@@ -0,0 +1,86 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+oppiaFoundationWebsite.controller('volunteerPage', [
+  '$scope', function(
+      $scope) {
+    var designTab = {
+      title: 'Design',
+      templateUrl: '/pages/volunteer/tabs_template/designTab.html'
+    };
+    var developmentTab = {
+      title: 'Development',
+      templateUrl: '/pages/volunteer/tabs_template/developmentTab.html'
+    };
+    var researchTab = {
+      title: 'Research',
+      templateUrl: '/pages/volunteer/tabs_template/researchTab.html'
+    };
+    var marketingTab = {
+      title: 'Marketing',
+      templateUrl: '/pages/volunteer/tabs_template/marketingTab.html'
+    };
+    var tabs = [designTab, developmentTab, researchTab, marketingTab];
+    $scope.tabs = tabs;
+    $scope.tabId = 0;
+    $scope.templateUrl = tabs[0].templateUrl;
+    $scope.onTabSelected = function(tabId) {
+      $scope.tabId = tabId;
+      $scope.templateUrl = tabs[tabId].templateUrl;
+    };
+    $scope.slideInterval = 0;
```

Comment:
Still no idea what these mean.

Reply:
These should not be here now that the carousel has its own directive. My bad!